tools/ossf_best_practices/gold_criteria.md

```diff
@@ -0,0 +1,248 @@
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307398661)
+
+# Quality
+
+## Coding standards
+
+> The project MUST document its code review requirements, including how code review is conducted, what must be checked, and what is required to be acceptable. (URL required)
+
+**Met. The process is documented: https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#reviewing-pull-requests**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307403399)
+
+> The project MUST have at least 50% of all proposed modifications reviewed before release by a person other than the author, to determine if it is a worthwhile modification and free of known issues which would argue against its inclusion
+
+**Met. Currently the repo is monitored against the OSSF Scorecard, where this is checked every 2 weeks in the Security Team regular meetings. See: https://github.com/nodejs/security-wg/blob/main/tools/ossf_scorecard/report.md**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+- [CII Best Practices: Two Person Review](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#two_person_review)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307402095)
+
+
+## Working build system
+
+> The project MUST have a [reproducible build](https://reproducible-builds.org/). If no building occurs (e.g., scripting languages where the source code is used directly instead of being compiled), select "not applicable" (N/A). (URL required)
+
+**Unmet**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+
+
+## Automated test suite
+
+
+> A test suite MUST be invocable in a standard way for that language.
+
+**Met**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+- [Team Discussion](https://github.com/nodejs/security-wg/pull/956#discussion_r1307404137)
+
+> The project MUST implement continuous integration, where new or changed code is frequently integrated into a central code repository and automated tests are run on the result. 
+
+**Met. https://ci.nodejs.org/**
+
+Context:
+- [CII Best Practices: Quality](https://github.com/coreinfrastructure/best-practices-badge/blob/main/docs/other.md#upgrade-quality-1)
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 90% statement coverage if there is at least one FLOSS tool that can measure this criterion in the selected language. 
```

Comment:
Check if we met the 90% percentage.

Reply:
I've no idea if https://app.codecov.io/gh/nodejs/node is able to show statement coverage (it's showing line coverage).
https://github.com/nodejs/node/actions/workflows/coverage-linux.yml?query=branch%3Amain is reporting [95.5% statement coverage in the most recent run](https://github.com/nodejs/node/actions/runs/7409195754/job/20158971473#step:8:19) for JS code (via c8, see the "Report JS" twisty) but unfortunately no summary/easily readable numbers for the C++ code.

Reply:
I created this issue to follow up with the discussion. https://github.com/nodejs/security-wg/issues/1188